src/coral_models/prepare_raw_data.py

```diff
@@ -0,0 +1,470 @@
+        )
+
+        # Update the filename in the recording metadata
+        recordings.loc[row_i, "filename"] = new_filename
+
+        # If the file is an .webm file, convert it to .wav
+        subprocess.run(
+            [
+                "ffmpeg",
+                "-y",  # overwrite output file if it exists
+                "-i",  # input file
+                str(filename),  # input file name
+                "-acodec",  # audio codec
+                "pcm_s16le",  # 16-bit signed integer PCM audio codec
+                "-ac",  # number of audio channels
+                "1",  # mono
+                "-f",  # force format
+                "wav",  # output format
+                new_filename,  # output file name
+            ],
+            stdout=subprocess.DEVNULL,
+        )
+
+        # Get the duration of each audio file
+        try:
+            new_filename_str = str(new_filename)
+            with contextlib.closing(wave.open(str(new_filename_str), "r")) as f:
+                frames = f.getnframes()
+                rate = f.getframerate()
+                duration = frames / float(rate)
+                if "conversation" in new_filename_str:
+                    conversation_duration += duration
+                else:
+                    read_aloud_duration += duration
+        except FileNotFoundError:
+            pass
+
+    # Make a dataframe with statistics about the data
+    data_stats = pd.DataFrame(
+        {
+            "Number of speakers": len(speakers),
+            "Number of sentences": len(sentences),
+            "Number of recordings": len(recordings),
+            "Read aloud duration (hours)": round(read_aloud_duration / 3600, 2),
+            "Conversation duration (hours)": round(conversation_duration / 3600, 2),
+            "Total duration (hours)": round(
+                (read_aloud_duration + conversation_duration) / 3600, 2
+            ),
+            "People over 50": len(speakers[speakers["age"] > 50]),
+            "People below 50 and above 18": len(
+                speakers[(speakers["age"] < 50) & (speakers["age"] > 18)]
+            ),
+            "People below 18": len(speakers[speakers["age"] < 18]),
+            "Number of woman": len(speakers[speakers["gender"] == "female"]),
+            "Number of men": len(speakers[speakers["gender"] == "male"]),
+            "Unique dialects": len(speakers["dialect"].unique()),
+        },
+        index=[0],
+    )
+
+    # Save the dataframes
+    data_stats.to_csv(output_path / "data_stats.csv", index=False)
+    speakers.to_csv(output_path / "speakers.csv", index=False)
+    sentences.to_csv(output_path / "sentences.csv", index=False)
+    recordings.to_csv(output_path / "recordings.csv", index=False)
+
+
+def correct_language(country: str) -> str:
+    """Converts a country name or alpha-2 code to a full country name
+
+    Args:
+        country (str): A country name or alpha-2 code
```

Comment:
What's an alpha-2 code? 2-letter language code?

Also, are we getting country codes or languages codes here? The function indicates language, but the docstring indicates country.